schema.sql

```diff
@@ -1,5 +1,9 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE ANIMALS (
+    ID INT, 
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that this column auto-increment every time a column is added as it is the primary key of your table
- [ ] [OPTIONAL] kindly add the primary key contraint